core/app/styles.scss

```diff
@@ -4,7 +4,7 @@ $images-path: '~@collab-ui/core/images';
 @import '~@collab-ui/core/scss/collab-ui';
 $side-nav-width: 240px;
 body {
-  background: $white;
+  background: $cui-white-100;
```

Comment:
Just a thought - should we prefix this with a variable in preparation for the md switch?

Reply:
Let's discuss for a future feature